ibis/expr/datatypes/core.py

```diff
@@ -665,15 +650,10 @@ class Floating(Primitive, Numeric):
     scalar = "FloatingScalar"
     column = "FloatingColumn"
 
-    @property
-    def largest(self):
-        """Return the largest type of floating point values."""
-        return float64
-
     @property
     @abstractmethod
     def nbytes(self) -> int:  # pragma: no cover
```

Comment:
It would be better to define `nbytes` on `dt.Numeric` but then we would need to have specific bitwidth decimals which could be worth doing based on the decimal precision, see https://clickhouse.com/docs/en/sql-reference/data-types/decimal